massgov/mayflower
# DP-9579 Remove raw filter from _patterns/03-organisms/by-author/event-listing-interactive.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from _patterns/03-organisms/by-author/event-listing-interactive.twig.

By removing the raw filter, the calendar icons in the date fields in the filter are not displayed, and the place holder text in the first field is not displayed from the JSON.

**With raw filter:**

<img width="726" alt="dp-9579_event-filter-filter" src="https://user-images.githubusercontent.com/9633303/43161309-beb67f32-8f55-11e8-96ec-ba6a05784cef.png">

**Without raw filter:**

<img width="761" alt="dp-9579_event-listing-no-filter" src="https://user-images.githubusercontent.com/9633303/43161337-ce0a1250-8f55-11e8-8a8b-7786a5937918.png">

The filter feature doesn't seem to be implemented in Drupal:

<img width="657" alt="dp-9579_event-listing2" src="https://user-images.githubusercontent.com/9633303/43161561-6a2f358e-8f56-11e8-9cf0-c62c7df10ba4.png">

The change doesn't affect the display of the event listing data which is from JSON where the filter was originally applied and the change was applied, but it seems to be better to merge`patternlab/disable_twig_autoescape` to ensure any data from JSON (considering event description data includes links) to be displayed properly.



Template use in Mayflower:
```
 ~/MassIT/mayflower/patternlab/ [feature/DP-9579_organisms_event-listing-interactive] grep -r 'event-listing-interactive.twig' styleguide/source
styleguide/source/_patterns/05-pages/event-listing.twig:  {% include "@organisms/by-author/event-listing-interactive.twig" %}
styleguide/source/assets/js/modules/eventListingInteractive.js:    const rawData = ma.eventListingInteractive[i]; // Data object created in @organisms/by-author/event-listing-interactive.twig
 ~/MassIT/mayflower/patternlab/ [feature/DP-9579_organisms_event-listing-interactive]
```

## Related Issue / Ticket

- [DP-9579](https://jira.mass.gov/browse/DP-9579)


## Steps to Test


1. Go to event listing page and check for any display issue in the event listing information.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
